src/core/execute.c

```diff
@@ -3216,11 +3216,18 @@ static int exec_child(
                 }
         }
 
-        if (context->cpu_set.set)
+        if (context->cpu_set.set || context->numa_policy.derive_cpu_affinity) {
```

Comment:
`derive_cpu_affinity` is very generic. Could this be something straightforward like `derive_cpu_affinity_from_numa_mask` or shorter?

Reply:
Will do, though I am not convinced that suffix is warranted given that the `bool` is part of the `NUMAPolicy` structure. IOW, placement of the variable provides enough context and we don't have to duplicate the information by including it in the variable name.